utils/db_setup/psqlInit.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+TEMP_DIR="/tmp/psql"
+
+mkdir -p ${TEMP_DIR}
+
+cp *.sql ${TEMP_DIR}
```

Comment:
why is this necessary?

Reply:
The `postgres` user doesn't have access to your home directory. Not going to tamper with people's file permissions in a script.

Reply:
got it, makes sense 👍

Reply:
If you want, I can make the script clean up the `/tmp` directly but I feel that is unnecessary.